### ISSUE TITLE
Comments Inconsistencies

### ISSUE THREAD

**Issue by sr4136** [bu-ist/responsive-framework#141]
After adding a new site with Responsi framework, the option `_bu_supports_comments` is not initialized, yet in `functions.php`, `'BU_SUPPORTS_COMMENTS', true` exists.

**Comment by toddmilliken** [comment]
That option value seems odd. We have to go through quite a few hurdles to enable comments here:

1. Enable in Settings > Discussion (normal WP functionality)
2. Ensure your pages individually have "Allow Comments" checked (normal WP functionality. Can be done via bulk edit from edit posts/pages screen)
3. Ensure your [post-type supports comments](https://developer.wordpress.org/reference/functions/register_post_type/) (normal WP functionality) 
4. Add a constant to set `BU_SUPPORTS_COMMENTS` to `true` if not defined already, OR add theme supports for `bu_comments`) (BU specific functionality)
5. Also set an option for `_bu_supports_comments` to be `1` or any value really that doesn't return a falsey value (BU specific functionality). 

As you can see, core WP gives us enough hurdles in steps 1-3 to enable comments for a specific post-type.

Then on top of that, steps 4 & 5 must be followed.

Can I ask why the option must exist in addition to adding theme supports? I don't see a way to set this in the admin, so this has to be a code change? The option name starts with an underscore which hides it from network options screen or /wp-admin/options.php for single site options screens.

Maybe we keep one of these approaches (must have theme supports bu comments) and do away with the other (must have option value set)? We'd have to run this by webteam folks to see if we can find more context as to why that option value must also exist.

**Comment by ashleykolodziej** [comment]
After extensive review, this is working **as intended**. Documentation here: https://developer.bu.edu/webteam/support/wordpress/enabling-comments-on-wordpress/

And I've verified this in the code in BU Comments. Basically, in order to enable comments, you have to contact IS&T. The UI does nothing, and when a new site is created, even though we set the default to not show comments, that checkbox (and site option) is completely disconnected from what actually happens.

I don't think this makes sense. I would rather see this rely on the WordPress UI, and let the theme decide whether or not to remove comment support through the filters WordPress gives us. This is easy enough to do, and we allow comment support by default anyway.

BU Site Manager can also set defaults. https://github.com/bu-ist/site-manager/issues/12 We could also restrict access to that settings page to only certain user roles, if that's not already what's happening. 

In other words, I'd like to see us deprecate and remove the BU Comments plugin entirely. https://github.com/bu-ist/bu-comments Removing this should not change our theme's default behaviors - everything already supports comments from a theme perspective. The biggest risk is that someone has mucked around in their site and tried to turn comments on, and we don't know it because the `_bu_supports_comments` option is hiding that behavior. One way to identify this would be to run a CLI command to identify if any sites have tried to turn comments on, and then see how many of those (hopefully very few sites) also have the `_bu_supports_comments` option set.
